image/image.go

```diff
@@ -87,6 +88,18 @@ func validate(w walker, refs []string, out *log.Logger) error {
 		if err := m.validate(w); err != nil {
 			return err
 		}
+
+		ml, err := findManifestList(w, d)
+		if err == nil {
+			if err := ml.validate(w); err != nil {
+				return err
+			}
+		} else {
+			if !strings.Contains(string(err), "manifestlist not found") {
```

Comment:
At the moment, the manifest list is OPTIONAL(https://github.com/opencontainers/image-spec/blob/master/manifest-list.md#oci-image-manifest-list-specification) , so we should not error out if there is no manifest list